src/platform/packages/shared/kbn-test/src/jest/resolver.js

```diff
@@ -77,6 +77,27 @@ module.exports = (request, options) => {
     });
   }
 
+  if (request === 'zod-to-json-schema') {
+    return resolve.sync('@finom/zod-to-json-schema', {
+      basedir: options.basedir,
+      extensions: options.extensions,
+    });
+  }
+
+  if (request === 'zod' || request === 'zod/v3') {
```

Comment:
after `zod@v4` we can remove it, but because they released 2 versions of lib `jest` couldn't resolve `.cjs` properly